docling/backend/asciidoc_backend.py

```diff
@@ -287,7 +287,7 @@ def _parse_title(self, line):
 
     #   =========   Section headers
     def _is_section_header(self, line):
-        return re.match(r"^==+", line)
+        return re.match(r"^==+\s+", line)
```

Comment:
Is this `\s+` or `\s*` (allowing for backward compatibility?)

Reply:
With `\s*` the space is optional and still keep to misidentify some blocks as header.

As an alternative the check could be in two steps and skip the case of `====` not followed by spaces. However, since a header section requires a mark followed by space and some text, then  I do not think it is necessary.

Reply:
OK, so this was by design, thanks!